client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
That doesn't compile, as the it has to be type_method. Error during 'go tool vet' is:
client/db_test.go:269: ExampleContext_Insecure refers to unknown identifier: Context